mini3d.c

```diff
@@ -875,7 +875,7 @@ void draw_box(device_t *device, float theta) {
 	device->transform.world = m;
 	transform_update(&device->transform);
 	draw_plane(device, 0, 1, 2, 3);
-	draw_plane(device, 4, 5, 6, 7);
+	draw_plane(device, 7, 6, 5, 4);

```

Comment:
```suggestion
	draw_plane(device, 7, 6, 5, 4); 
```
此处改为draw_plane(device, 5, 4, 7, 6);，所有的顶点都为顺时针绕序，上面的顶点数据就不用改了。